docs/en/compute/Compute-HBase.md

```diff
@@ -14,9 +14,10 @@ that you can easily store HBase tables into Alluxio at various storage levels.
 
 ## Prerequisites
 
-* An Alluxio cluster has been set up and is running according to either 
-[Local Mode]({{ site.baseurl }}{% link en/deploy/Running-Alluxio-Locally.md %}) or 
-[Cluster Mode]({{ site.baseurl }}{% link en/deploy/Running-Alluxio-On-a-Cluster.md %}).
+* An Alluxio cluster has been set up and is running according to 
```

Comment:
How about shortening this to "Alluxio has been set up and is running"? I don't see the point of calling out a couple specific ways that Alluxio might be deployed.

Reply:
Agree!
Changed  to `Alluxio has been set up and is running`
Thanks!